recipes/configs/dev/multimodal_generation.yaml

```diff
@@ -0,0 +1,38 @@
+# Config for running the InferenceRecipe in generate.py to generate output from an LLM
+#
+# To launch, run the following command from root torchtune directory:
+#    tune run dev/generate_v2 --config multimodal_generation
+
+# Model arguments
+model:
+  _component_: torchtune.models.flamingo_11b
+
+# Transform arguments
+transform:
+  _component_: torchtune.models.flamingo_transform
+  tokenizer_path: /tmp/flamingo/tokenizer.model
+  max_seq_len: 8192
+
+# Checkpointer
+checkpointer:
+  _component_: torchtune.training.FullModelMetaCheckpointer
+  checkpoint_dir: /tmp/flamingo
+  checkpoint_files: ["consolidated.pth"]
+  output_dir: /tmp/flamingo
+  model_type: FLAMINGO
+
+device: cuda
+dtype: bf16
+seed: 1234
+log_level: INFO
+
+# Generation arguments
+prompt:
+  system: You are a helpful assistant who responds like the author Shakespeare.
+  user:
```

Comment:
Does this support multiple image/text entries or just preprend image?

Reply:
prepend